MIP/mip-53/README.md

```diff
@@ -0,0 +1,66 @@
+# MIP-53: Conventions for Proposing Progressive L2 Models
+- **Description**: Introduces conventions for proposing progressive L2 models.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Reviewer**: [Andreas Penzkofer](mailto:andreas.penzkofer@movementlabs.xyz)
+- **Desiderata**: $\emptyset$
+
+## Abstract
+
+Movement's L2 system will progress through various changes during its life, and each stage is presented through a new L2 model (which we call **progressive L2 model**). Thus a given progressive L2 model describes as a particular stage in the evolution of the L2 system.  A given progressive L2 model stage is defined through the sum of its improvements and it may encompass components that are not directly related to each other. 
+
+We propose a set of conventions including naming, formatting, and related standards to assist reviewing proposals for said models. Finally a collection point of these terms to provide an overview
+
+## Motivation
+
+In order to facilitate the development of L2 systems, it is important to have a clear and consistent way to propose and review models for these systems. This MIP aims to provide a set of conventions for proposing progressive L2 models that will help to ensure that proposals are clear, consistent, and easy to review.
+
+## Specification
+Progressive L2 models should adhere to conventions of the following forms:
+
+- **naming**: apply a standard naming format.
+- **acknowledgement of standards**: acknowledge the conventions of this MIP.
+- **summary table**: a table succinctly noting key features of the model.
+- **statement towards progression**: a statement of how the model suits a progressive approach to L2 design and release.
+- **pros and cons**: a list of pros and cons.
+
+### Naming
+All proposed progressive L2 models MUST adopt a name of the form "The [Location] Model" where [Location] is a location where the proposal was drafted or a similarly symbolic location. This name should be the title of the associated MIP. 
+
+In text, the model SHOULD be referred to as "[Location] Model," that is, the name of the model in uppercase. 
```

Comment:
Do you mean initial caps, rather than uppercase, for the name of the model?

For example, the Amherst Model is what you intend, right?

Reply:
yes title case. thx